openvdb/openvdb/python/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = [
+    "scikit_build_core==0.2.1",
+    "pybind11==2.10.3",
+    "oldest-supported-numpy"
+]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "openvdb"
+version = "10.0.1"
+description= "Python bindings for OpenVDB: sparse volume data structure and tools."
+dependencies = [
+    "oldest-supported-numpy",
+]
+readme = "README.md"
+authors = [
+  { name = "OpenVDB Developer Team", email = "openvdb-dev@lists.aswf.io" },
+]
+requires-python = ">=3.7"
+classifiers = [
+  "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+]
+
+[project.urls]
+homepage = "https://www.openvdb.org/"
+documentation = "https://www.openvdb.org/documentation/doxygen"
```

Comment:
I'm not an OpenVDB user, so sorry if I misunderstand something here, but wouldn't https://www.openvdb.org/documentation be a better URL than the C++ docs?

Reply:
Sure - this was just the link from the `README.md`, but I can point to the docs page one level up.